plugins/easyimage/styles/easyimage.css

```diff
@@ -43,6 +46,15 @@ The outline is not a part of the element's dimensions, we have to use a border a
 	max-width: 25%;
```

Comment:
With that style applied to `alignCenter` large images will be reduced to 25% of width.

![image](https://user-images.githubusercontent.com/5353898/35632934-5888f5a4-06a8-11e8-8f37-52291999e614.png)

Reply:
Actually I find this entire block now very, very little readable. I think it would be better idea to split it into 3 groups (left, right, center). Even if it would cost us repeating `max-width` / `float` I feel that would be more readable.

Reply:
I've limit center styles to 90% (just to differentiate them from `full` styles).